docker-compose.yml

```diff
@@ -62,11 +62,22 @@ services:
     image: memcached
 
   mysql:
-    build: ./docker/images/mysql
+    image: mysql:5.6
     environment:
       - MYSQL_USER=kuma
       - MYSQL_PASSWORD=kuma
       - MYSQL_DATABASE=developer_mozilla_org
+      - MYSQL_ROOT_PASSWORD=kuma
+    command:
+      - mysqld
+      - --character-set-server=utf8
+      - --collation-server=utf8_general_ci
+      - --innodb-buffer-pool-size=1G
+      - --innodb-flush-log-at-trx-commit=0
+      - --innodb-log-file-size=1G
+      - --max-allowed-packet=100M
+    expose:
+      - "3306"
```

Comment:
Good eye @safwanrahman. That wasn't needed locally, but I was trying to get TravisCI to work, which gives the [error](https://travis-ci.org/mozilla/kuma/jobs/319495339#L4268:

> Host '172.18.0.9' is not allowed to connect to this MySQL server

I thought this might work because [expose](https://docs.docker.com/compose/compose-file/#expose) will expose ports without publishing them to the host machine, so they are accessible to linked services. However, the error is from MySQL, so the issue is not that the port is blocked, but that MySQL is refusing the credentials.

I'll remove it from the commit, and add a "Try things to get TravisCI working" commit.